sentry_streams/src/callers.rs

```diff
@@ -0,0 +1,50 @@
+use crate::routes::RoutedValue;
+use pyo3::prelude::*;
+use sentry_arroyo::types::Message;
+
+/// Executes a Python callable with an Arroyo message containing Any and
+/// returns the result.
+pub fn call_python_function(
```

Comment:
Is this meant to be a generic wrapper around python calls? If so, maybe this should be in a mod call "python"?

Reply:
I meant this more as a place to keep the different ways to call back into python function, so not everything python. Like python exposed data structures are in the modules where they belong to like `routes.rs` rather than in a general `python` module.